src/main/java/ua/danit/final_project/controllers/TaskController.java

```diff
@@ -11,7 +10,7 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.multipart.MultipartFile;
-import ua.danit.final_project.configuration.StaticCollection;
+import ua.danit.final_project.configuration.SessionAware;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.configuration.SessionAware'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=131&issues=AWY0HPG5M63is4RD-5Te&open=AWY0HPG5M63is4RD-5Te)

[ISSUE_KEY]: AWY0HPG5M63is4RD-5Te